### PR TITLE
Remove Remaining Redis References

### PR DESCRIPTION
**Pull request by phil-lopreiato**
Just some assorted tooling/scripts. We can merge this once we finally turn off the GCE instances, just in case we need to revert before then.

### CODE REVIEW

File: ops/shell/memorystore_shell.sh [hunk 1]
```diff
@@ -1,91 +0,0 @@
-#! /bin/bash
```
Comment (ZachOrr) [inline ops/shell/memorystore_shell.sh:1]: Can we drop this section from the maintainer runbook as well? Maybe with a quick tl;dr of what is cached, and when it'll be cleared?

**Comment by ZachOrr** [comment]
Can we also remove `pyyaml`? I'm going to drop the `deploy_queues` script, which is the only other dependent of that library. One less library to install when bootstrapping

**Comment by phil-lopreiato** [comment]
Yeah, I can drop that too

File: ops/shell/memorystore_shell.sh (hunk 1 above)
Comment (ZachOrr) [reply ops/shell/memorystore_shell.sh:1]: We can actually remove the `Memorystore (redis)` section of the `GAE Firebase Setup` page as well!

**Pull request merged**